README.md

```diff
@@ -172,6 +172,19 @@ data on your cluster will be lost:
 
     muchos terminate
 
+## Automatic shutdown of EC2 clusters
+
+With the default configuration, EC2 clusters will not shutdown automatically after a delay and the default
+shutdown behavior will be stopping the node.  If you would like your cluster to terminate after 8 hours,
+set the following configuration in `muchos.props`:
+
+```
+shutdown_delay_minutes = 480
+shutdown_behavior = terminate
+```
+
+If you decide later to cancel the shutdown, run `muchos cancel_shutdown`.
```

Comment:
Will this still leave the shutdown behavior as terminate even if the config was changed to stop?